pkg/common/configs/configvalidator.go

```diff
@@ -489,7 +489,7 @@ func checkLimit(limit Limit, existedUserName map[string]bool, existedGroupName m
 		}
 
 		if existedUserName[name] {
```

Comment:
We need some spell check here as well... let's do it in a different JIRA.

Reply:
Sure.